pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -537,45 +538,45 @@ message ContainerConfig {
     optional bool stdin_once = 13;
     optional bool tty = 14;
 
-    // Linux contains configuration specific to Linux containers.
+    // Configuration specific to Linux containers.
```

Comment:
maybe change this to `Linux specific configuration` (highlight linux instead of containers).